entrypoint.sh

```diff
@@ -79,8 +79,9 @@ if [ "$ADD_REPOSITORIES" != "" ]; then
   helm repo update
 fi
 
-cd ${CHART_FOLDER}
+cd "${CHART_FOLDER}"
 helm lint .
-helm package . ${REGISTRY_APPVERSION} ${REGISTRY_VERSION} ${UPDATE_DEPENDENCIES}
-helm inspect chart *.tgz
-helm cm-push *.tgz ${REGISTRY_URL} ${REGISTRY_USERNAME} ${REGISTRY_PASSWORD} ${REGISTRY_ACCESS_TOKEN} ${FORCE}
+OUT_DIR=$(mktemp -d)
+eval $(echo helm package . -d "${OUT_DIR}" "${REGISTRY_APPVERSION}" "${REGISTRY_VERSION}" "${UPDATE_DEPENDENCIES}")
```

Comment:
What is this line supposed to do? Why would it not be possible to write this as

```suggestion
helm package . -d "${OUT_DIR}" "${REGISTRY_APPVERSION}" "${REGISTRY_VERSION}" "${UPDATE_DEPENDENCIES}"
```

instead? Or could

```suggestion
sh -c 'helm package . -d "${OUT_DIR}" ${REGISTRY_APPVERSION} ${REGISTRY_VERSION} ${UPDATE_DEPENDENCIES}'
```

be a way?

be a way?

Reply:
Or even shorter?

```suggestion
helm package . -d ${OUT_DIR} ${REGISTRY_APPVERSION} ${REGISTRY_VERSION} ${UPDATE_DEPENDENCIES}
```

What is the observed failure when approaching it this naively?

Reply:
There's also

```suggestion
sh -c "helm package . -d '${OUT_DIR}' '${REGISTRY_APPVERSION}' '${REGISTRY_VERSION}' '${UPDATE_DEPENDENCIES}'"
```

which could probably help to work around the issue with escaping special characters.